packages/utils/src/modules-sdk/loaders/container-loader-factory.ts

```diff
@@ -0,0 +1,162 @@
+import {
```

Comment:
comment: We will need to be very clear about the naming of models and how that relates to repos. 

I spent a couple of hours debugging why a repo wasn't correctly injected, only to find out the repo names can't be different from the model names.

Also, the "failed injection" error was swallowed somewhere, which is what made it difficult to debug. Maybe something we could look into?

Reply:
Yeah if the model is name SomethingElse then the repo will be somethingElseRepository in the container. The error should be visible in the log, the load-internal from the module SDK should restitute you the message. Can you share the logs you received, if it is in the tests you have to go back up to the top of the logs and it is somewhere there

Reply:
So currently, the naming are the same as they were normally